### DISCUSSION TITLE
SlingerPlayer.exe working under WINE/Linux

### DISCUSSION THREAD

**Discussion by knoppix1337** [GerryDazoo/SlingerPlayer#7] [Show and tell]
![Screenshot from 2023-03-30 09-36-53](https://user-images.githubusercontent.com/19241297/228875241-ae2d2b2c-3a60-456d-8cd0-8e73d871a680.png)

I put the SlingerPlayer.exe along with the *.dll files in a folder and it opened it with WINE (Wine Is Not an Emulator) https://en.wikipedia.org/wiki/Wine_(software) on Linux and it works!

From the screenshot above, whatever the icons or characters for play/pause mute/unmute and fullscreen are showing up as square boxes (although the tooltips do work), If they are a font, it may be missing on my system. Let me know and I'll report back what solution I find that works.

Even the "Always on Top" works.

The remote button only brings up a blank window but I am unsure how it's meant to normally work (also I have my port number set as 8090 because I get almost no URL "drive by's" in the servers sling.log as compared to 8080).

The aspect ratio defaults to "Src" and doesn't remember the last setting.

My browser window (on Linux or my phone) that I normally use for the "remote" works fine, and the lag time is like half of what it was, or better.

I also like the programs "S" icon, very creative.

Thanks, SlingerPlayer is a very useful addition to the overall project.

- Eric

**Comment by GerryDazoo** [comment]
Good news that it's usable.
The font it is using is Tahoma.
You need a browser installed for the code to use. I think this will work if it's Wine-able
https://developer.microsoft.com/en-us/microsoft-edge/webview2/
Give it a try and let me know.
The icon came from a helpful person on Discord.
If your phone is Andriod, SlingerPlayer has an integrated remote like the Windows version.

**Comment by GerryDazoo** [comment]
An update, the missing buttons are Unicode Emojis 
var FScreen : String = #$2725;
  var Play : String =  #$25B6#$FE0F;
  var Stop : String = #$1F6D1;
  var Muted : String = #$1F507;
  var unMuted : String = #$1F509;
  var VolUp : String = #$1F50A;

**Reply by bradut** [reply]
@GerryDazoo: I have the same issue when the player is running in Windows 7. 
How can I use these codes to display symbols on the Slinger Player buttons?
